features/in_onderzoek.feature

```diff
@@ -127,33 +127,32 @@ Functionaliteit: in onderzoek
       Gegeven het systeem heeft een persoon met de volgende gegevens
       | naam                | waarde    |
       | burgerservicenummer | 555550001 |
-      En de nationaliteit heeft volgende 'inOnderzoek' gegevens
-      | naam                          | waarde   |
-      | aanduidingGegevensInOnderzoek | 047310   |
-      | datumIngangOnderzoek          | 20220307 |
+      En de persoon heeft een 'nationaliteit' met de volgende gegevens
+      | naam                                     | waarde   |
+      | aanduiding gegevens in onderzoek (83.10) | 047310   |
+      | datum ingang onderzoek (83.20)           | 20220307 |
       Als personen wordt gezocht met de volgende parameters
-      | naam                | waarde                              |
-      | type                | RaadpleegMetBurgerservicenummer     |
-      | burgerservicenummer | 555550001                           |
-      | fields              | burgerservicenummer,nationaliteiten |
-      Dan heeft de persoon met burgerservicenummer '555550001' GEEN 'inOnderzoek' gegevens
-      En heeft de nationaliteit GEEN 'inOnderzoek' gegevens
+      | naam                | waarde                          |
+      | type                | RaadpleegMetBurgerservicenummer |
+      | burgerservicenummer | 555550001                       |
+      | fields              | nationaliteiten                 |
+      Dan heeft de response een persoon met een 'nationaliteit' met de volgende gegevens
+      | naam | waarde                |
+      | type | NationaliteitOnbekend |
```

Comment:
Op basis waarvan krijgt hier 'type' de waarde 'NationaliteitOnbekend'?

Reply:
ik denk ook dat je ten minste de nationaliteit moet opgeven bij Gegeven

Reply:
welke nationaliteit moet dan worden opgegeven? In de initiele scenario is ook geen nationaliteit opgegeven en uit de scenario titel kan ik niet afleiden wat voor nationaliteit een persoon met EU-persoonsummer heeft

Reply:
issue voor gemaakt: #1202